Kumi.Game/Input/GameplayKeybindContainer.cs

```diff
@@ -0,0 +1,22 @@
+﻿using osu.Framework.Input.Bindings;
+
+namespace Kumi.Game.Input;
+
+public partial class GameplayKeybindContainer : KeyBindingContainer<GameplayAction>
```

Comment:
Sole reason I'm not using `RealmBackedKeyBindingContainer` is because it simply does not work, and I presume this is because we'd have to apply the defaults using the `KeybindStore` somewhere, which we'll need a better way to do that in the future.

Reply:
you could have just made a seperate pr do just that